packages/flutter/lib/src/material/floating_action_button_location.dart

```diff
@@ -419,8 +419,12 @@ mixin FabTopOffsetY on StandardFabLocation {
   /// the transition between the [Scaffold.appBar] and the [Scaffold.body].
   @override
   double getOffsetY(ScaffoldPrelayoutGeometry scaffoldGeometry, double adjustment) {
-    final double fabHalfHeight = scaffoldGeometry.floatingActionButtonSize.height / 2.0;
-    return scaffoldGeometry.contentTop - fabHalfHeight;
+    if (scaffoldGeometry.contentTop > scaffoldGeometry.minViewPadding.top) {
```

Comment:
We're ignoring the adjustment parameter here, same as before (@pennzht - why is that?). Maybe update the API doc?

Reply:
According to the docs, the Y offset is not affected by the adjustment for the mini Top positions. It looks like only the X is adjusted.
https://github.com/flutter/flutter/blob/9322c80825bb582954410408de39c0ef90b395ec/packages/flutter/lib/src/material/floating_action_button_location.dart#L40

Reply:
We're providing API doc for this override, so it would be helpful to reiterate the point about ignoring the adjustment parameter here.